cmd/cadvisor_test.go

```diff
@@ -40,6 +40,12 @@ func TestUdpMetricsAreDisabledByDefault(t *testing.T) {
 	assert.True(t, ignoreMetrics.Has(container.NetworkUdpUsageMetrics))
 }
 
+func TestWssMetricIsDisabledByDefault(t *testing.T) {
+	assert.True(t, ignoreMetrics.Has(container.WssMetric))
```

Comment:
What is purpose of this assertion?

Reply:
checking if wss metric is on the list of metrics  which are disabled by default